post-processing/post-process.py

```diff
@@ -145,8 +146,13 @@ def main():
     file_path = args.path
     merged_file_name = args.outputname
     input_name = args.inputname
+    decimals = args.decimals
     mpiranks = read_mpi_ranks(file_path, input_name)
 
+    if decimals >= 4:
+
+        print("WARNING: Calculated values may not be accurate or representative at higher precisions and may not display correctly")
```

Comment:
Is this due to the post processing script of Vernier itself? If it's because of Verniers precision of output then we should say.